tests/models/test_teacher_forcing_logits.py

```diff
@@ -14,8 +14,11 @@ def test_method_get_teacher_forced_logits_for_encoder_decoder_model():
     transformers = pytest.importorskip("transformers")
 
     name = "hf-internal-testing/tiny-random-BartModel"
-    tokenizer = transformers.AutoTokenizer.from_pretrained(name)
-    model = transformers.AutoModelForSeq2SeqLM.from_pretrained(name)
+    try:
+        tokenizer = transformers.AutoTokenizer.from_pretrained(name)
+        model = transformers.AutoModelForSeq2SeqLM.from_pretrained(name)
+    except OSError:  # OSError includes requests.exceptions.RequestException
```

Comment:
Why don't we except only `RequestException`?

Reply:
OK, updated accordingly.

I initially kept it as an `OSError` as I think I once encountered an exception that did not inherit from `RequestException` when testing locally, but I can't reproduce. I think you're right, might as well be specific.